share/commands/technique-migrate

```diff
@@ -0,0 +1,73 @@
+#!/bin/sh
+
+#. "${BASEDIR}/../lib/api_call.sh"
+. "/opt/rudder/share/lib/api_call.sh"
+
+TECHNIQUES_DIRECTORY="${CONFIGURATION_DIRECTORY}/technique"
+TOKEN=""
+DISPLAY_COMMAND=false
+API_URL="https://localhost/rudder"
+
+while getopts "iIvdqct:n:" opt; do
+  case $opt in
+    i|I|v|d)
+      DISPLAY_COMMAND=true
+      ;;
+    t)
+      TOKEN="${OPTARG}"
+      ;;
+    n)
+      TECHNIQUE_NAME="${OPTARG}"
+      ;;
+    c)
+      clear_colors
+      ;;
+  esac
+done
+# to keep the argument as $1
+shift  $((OPTIND-1))
+
+if [ -z "${TECHNIQUE_NAME}" ]
+then
+  echo "Usage: rudder technique migrate [-iIvd] -t token -n technique_name [target_technique_version]"
+  exit 1
+fi
+
+TECHNIQUE_VERSION="$1"
+if [ -z "${TECHNIQUE_VERSION}" ]
+then
+  TECHNIQUE_VERSION="latest"
+fi
+
+if [ "${TECHNIQUE_VERSION}" == "latest" ]
+then
+# Find the newest technique version
+  TECHNIQUE_VERSION=$(complete_api_call "${API_URL}/api/latest/techniques" "${TOKEN}" "GET" "|jq -r '.data | .techniques[] | select( .name == \"${TECHNIQUE_NAME}\") | .versions[]' | tail -n1" "${DISPLAY_COMMAND}")
```

Comment:
I this the pipe should be used out of the api call function, it makes this more readable and less error prone